### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lorenzolewis**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,10 @@
...
 
-[![status](https://img.shields.io/badge/Status-Beta-green.svg)](https://github.com/tauri-apps/tauri)
-[![Chat Server](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/SpmNs4S)
-[![devto](https://img.shields.io/badge/blog-dev.to-black.svg)](https://dev.to/tauri)
-
-![](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library
-)
+[![status](https://img.shields.io/badge/Status-Stable-green.svg)](https://github.com/tauri-apps/tauri)
+[![Chat Server](https://img.shields.io/badge/chat-discord-7289da.svg)](https://discord.gg/SpmNs4S)
+![](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)
```
Comment (amrbashir) [inline README.md]: is this label supposed to be something else other than `test-library`?

**Review by amrbashir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amrbashir) [reply README.md]: should we delete the badge or change the label to just `tests`?

Comment (amrbashir) [reply README.md]: Also I think this badge exists in the main tauri repo

Comment (lorenzolewis) [reply README.md]: I'm not sure for the one on line 7, I just moved it up a line but it was already there. I'm assuming our testing will need to be redone after the re-write so maybe we keep as-is and update it when we update tests?

Comment (lorenzolewis) [reply README.md]: Oh, just realised it's pointing to the main repo, let me just remove that label from here

**Comment by lorenzolewis** [comment]
Added a few more changes, I'll put in a PR against the main repo for these as well

**Review by amrbashir — APPROVED** [review]

**Pull request merged**